index.html

```diff
@@ -285,9 +285,17 @@ <h2>Coordination</h2>
       <div>
         <h3 id="w3c-coordination">W3C Groups</h3>
 
-        <p>To promote interoperability with the overall web platform, and for all Recommendation-track documents, the
-        Working Group SHALL seek <a href="https://www.w3.org/Guide/Charter.html#horizontal-review">horizontal
-        review</a> with the W3C Horizontal Groups listed below.</p>
+
+
+        <p>For all specifications, this Working Group will seek <a
+          href="https://www.w3.org/Guide/process/charter.html#horizontal-review">horizontal review</a> for
+          accessibility, internationalization, performance, privacy, and security with the relevant Working and
+          Interest Groups, and with the <a href="https://www.w3.org/2001/tag/"
+          title="Technical Architecture Group">TAG</a>. Invitation for review must be issued during each
+          major standards-track document transition, including <a href="https://www.w3.org/Consortium/Process/#RecsWD"
+          title="First Public Working Draft">FPWD</a> and at least 3 months before
+          <a href="https://www.w3.org/Consortium/Process/#RecsCR" title="Candidate Recommendation">CR</a>,
+          and should be issued when major changes occur in a specification.</p>
 
         <dl>
           <dt>
```

Comment:
Add i18n group explicitly, even though it is already included in the list of horizontal groups, to address https://github.com/w3c/strategy/issues/177#issuecomment-499559628.